ads/_config.js

```diff
@@ -202,6 +202,10 @@ export const adConfig = {
     ],
   },
 
+  'baidu': {
+    prefetch: 'https://dup.baidustatic.com/js/dm.js',
+  },
```

Comment:
could you please implement the render-start API?

full [guideline](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request)